tests/e2e/support/helpers.js

```diff
@@ -197,6 +197,16 @@ export function assertSnackbarError(message, snackName = 'game') {
     .click();
 }
 
+export function assertCustomSnackBar(message, snackName = 'game') {
+  cy.get(`[data-cy=${snackName}-snackbar] .v-snackbar__wrapper`)
+    .should('be.visible')
+    .should('have.class', 'bg-surface-1')
+    .find(`.v-snackbar__content`)
+    .should('contain', message)
+    .get('[data-cy=close-snackbar]')
+    .click();
+}
```

Comment:
Instead of defining a new function for this, we should update the existing `assertSnackBarError()` to generalize by making an optional argument for the color and renaming it to simply `assertSnackBar()`